src/Query.php

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
```

Comment:
Correct Joomla doc blocks here please (and in the model.php and test files too :) )